src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -26,6 +26,7 @@
     private String clientName = "";
```

Comment:
Just a matter of consistency with other implementations, even if the names are not the best ones. Following your reasoning, it should be `clientNames` instead of `clientName`, shouldn't it?
That said, it's not a big deal.